docs/configuration.md

```diff
@@ -0,0 +1,267 @@
+| `hsts`                        |                                                   | `{"enable": true, "maxAgeSeconds": 31536000, "includeSubdomains": true, "preload": true}`                                                  | [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) options to use with HTTPS (default is the example value, max age is a year)                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+|                               | `CMD_HSTS_ENABLE`                                 | ` true`                                                                                                                                    | set to enable [HSTS](https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security) if HTTPS is also enabled (default is ` true`)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| ----------------------------- | ------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
This line seems to break the table rendering, at least in Githubs preview.
Is a horizontal line even necessary here?

Reply:
Note, this is a typo. I'll fix that.

Reply:
Fixed in e46d520ca234c05cfe2431c5b7ab61b96194962e.